src/main/java/com/epam/petclinic/order/config/OAuth2FeignRequestInterceptor.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.petclinic.order.config;
+
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.oauth2.client.OAuth2ClientContext;
+import org.springframework.util.Assert;
+
+public class OAuth2FeignRequestInterceptor implements RequestInterceptor {
+
+    /**
+     * The logger instance used by this class.
+     */
+    private static final Logger LOGGER = LoggerFactory.getLogger(OAuth2FeignRequestInterceptor.class);
+
+    /**
+     * The authorization header name.
+     */
+    private static final String AUTHORIZATION_HEADER = "Authorization";
```

Comment:
Could you please also check whether there is a constant in Spring with this value?